### DISCUSSION TITLE
annotation list refresh rate

### DISCUSSION THREAD

**Discussion by Emilien1** [grafana/grafana#47485] [Dashboards, Panels & Visualizations]
Hello,

I am using Grafana community Version 8.4.5.

The annotation list panel does not refresh as per dashboard refresh rate.
This would be useful to allow dashboard refresh to affect annotation list panel, especially in case of back end scripts pushing annotations.

Is there a plan to synchronize annotation list panel refresh rate with global dashboard refresh rate ?

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
